src/node/networking.ml

```diff
@@ -64,6 +64,11 @@ module Block_by_hash_spec = struct
   type response = Block.t option [@@deriving yojson]
   let path = "/block-by-hash"
 end
+module Block_by_level_spec = struct
+  type request = { level : int64 } [@@deriving yojson]
+  type response = Block.t option [@@deriving yojson]
+  let path = "/block-by-level"
+end
```

Comment:
It's a bit late since we already have `Block_by_hash_spec`, but why do these end with `spec` while others don't?